packages/lru/docs/api/classes/TimeLruCache.md

```
@@ -0,0 +1,460 @@
+
+// 🖖 Iterate over the cache entries
+for (const [key, value] of lru) {
+  results.push([key, value]);
+}
+
+expect(results).toStrictEqual([
+   ['key3', 'value3'], // 👈  Least recently used first
+   ['key2', 'value2'], // 👈  Most recently used last
+]);
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)